### PR TITLE
MoonGuard articles and Orion app

### PR DESCRIPTION
**Pull request by rpereira-tae**
Hi 👋🏻, we would like to share three articles from our blog and an app called "Orion". Below are the links:
 
1. [Managing State in Desktop Applications with Rust and Tauri](https://blog.moonguard.dev/manage-state-with-tauri)
2. [Setting up Actix Web in a Tauri App](https://blog.moonguard.dev/setting-up-actix-in-tauri)
3. [How to use local SQLite database with Tauri and Rust](https://blog.moonguard.dev/how-to-use-local-sqlite-database-with-tauri)
4. [Orion](https://github.com/taecontrol/orion)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [ ] If the project is closed source add the `![closed source]` badge after the `(link)` but before the `-`.
- [ ] If the project/article is non-free or paywalled add the `![paid]` badge after the `(link)` but before the `-`.
- [x] Add entries alphabetically to the most appropriate category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [ ] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

**Comment by FabianLars** [comment]
Thanks for your contribution! We love to see more blog posts!

Could you move the app into a separate PR please?
grafik
(technically that means the articles need to be in separate prs too but that's a bit too picky even for me :D )

**Comment by rpereira-tae** [comment]
Thank you for responding so quickly. We are enjoying developing with Tauri!

Yes, sure. I will move the app to another PR. 🙏🏻

File: README.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 ### Articles
 
 - [Getting Started Using Tauri Mobile](https://medium.com/p/6f90de5b098) ![paid] - Ed Rutherford outlines how to create a mobile app with Tauri
+- [How to use local SQLite database with Tauri and Rust](https://blog.moonguard.dev/how-to-use-local-sqlite-database-with-tauri) - Guide to setup and use SQLite database with Tauri and Rust.
+- [Managing State in Desktop Applications with Rust and Tauri](https://blog.moonguard.dev/manage-state-with-tauri) - How to share and manage any kind of state globally in Tauri apps.
+- [Setting up Actix Web in a Tauri App](https://blog.moonguard.dev/setting-up-actix-in-tauri) - How to setup a HTPP server with Tauri and Actix Web.
```
Comment (FabianLars) [inline README.md]:
```suggestion
- [Setting up Actix Web in a Tauri App](https://blog.moonguard.dev/setting-up-actix-in-tauri) - How to setup a HTTP server with Tauri and Actix Web.
```

**Review by FabianLars — APPROVED** [review]
Thank you!

**Pull request merged**